### DISCUSSION TITLE
[@types/inquirer] rxjs embedded as a dependency

### DISCUSSION THREAD

**Discussion by nfroidure** [DefinitelyTyped/DefinitelyTyped#63812] [Issues with a @types package]
I recently found out that `@types/inquirer` depends on RXJS which is very fat 12Mo. Is it really necessary?

[a comment by an automated account is not shown]

**Comment by neurolag** [comment]
As far as I can tell, it is - in order to represent `inquirer`s types properly.
`rxjs` ships its own type declarations and there is no dedicated `@types`-package for it.

Though, I think it might be a good idea to have a DT maintainer share their opinion on this matter.
